contracts/adapters/dmm/README.md

```diff
@@ -0,0 +1,4 @@
+# Notice
```

Comment:
we can add this information to the repo's wiki in order to keep everything in one place